src/F2_autoreport.py

```diff
@@ -11,6 +11,30 @@
 import base64
```

Comment:
meta_comment1: "SS" in "CSS" already stands for Style Sheet :)

meta_comment2: nice commit message :)